client/src/pages/CustomerProfile.js

```diff
@@ -62,20 +64,41 @@ const CustomerProfile = () => {
   //Dummy profile data for now
   const { userInfo } = useContext(UserContext);
 
-  const userData = {
+  const user = {
     name: "Christine Wilson",
     city: "Toronto, Canada",
     about:
       "Hi everyone! I'm a foodie and I love to eat healthy and tasty meals. Also I'm a mom of two beautiful babies.",
-    favCuisines: ["Japanese", "Chinese", "Mediterranean", "Thai"],
+    cuisines: ["Japanese", "Chinese", "Mediterranean", "Thai"],
     location: {
       lat: 43.6398,
       lng: -79.39793,
     },
   };
   const classes = useStyles();
+  const [open, setOpen] = useState(false);
+  const handleClose = (value) => {
+    setOpen(false);
+  };
+  const handleClickOpen = (e) => {
+    setOpen(true);
+  };
```

Comment:
If you don't need to do anything else on click you could call setOpen in in anonymous arrow function